perf_test/blas/blas/KokkosBlas_trtri_perf_test.hpp

```diff
@@ -320,19 +329,23 @@ trtri_args_t __do_setup(options_t options, matrix_dims_t dim) {
   uint64_t seed = Kokkos::Impl::clock_tic();
   Kokkos::Random_XorShift64_Pool<execution_space> rand_pool(seed);
   decltype(dim.a.m) min_dim = dim.a.m < dim.a.n ? dim.a.m : dim.a.n;
+  typename vta::HostMirror host_A;
   STATUS;
 
   trtri_args.uplo = options.blas_args.trtri.trtri_args.c_str()[0];
   trtri_args.diag = options.blas_args.trtri.trtri_args.c_str()[1];
   trtri_args.A    = vta("trtri_args.A", options.n, dim.a.m, dim.a.n);
+  host_A = Kokkos::create_mirror_view(trtri_args.A);
```

Comment:
@e10harvey you should be able to create `host_A` via `auto` type deduction, I don't think the declaration at line 332 above is necessary unless you prefer this pattern